tests/syscall/hostfs/enc/enc.c

```diff
@@ -18,6 +19,19 @@ void test_hostfs(const char* tmp_dir)
         exit(1);
     }
 
+    /* Mount with a relative source should fail */
+    if (mount(".", "/", OE_HOST_FILE_SYSTEM, 0, NULL) == 0)
```

Comment:
Is there a test to ensure that after we mount an absolute path, fopen with relative path succeeds? I am just trying to figure out if we have tests to cover the recently reported issue with fopen

Reply:
I added a test to `tests/syscall/fs` to check this